src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/arg_max_min/arg_max_min_kernel_axis.cpp

```diff
@@ -93,9 +93,14 @@ KernelsData ArgMaxMinKernelAxis::GetKernelsData(const Params& params, const opti
     auto jit = CreateJit(kernelName, cldnn_jit, entry_point);
 
     auto& kernel = kd.kernels[0];
-    FillCLKernelData(kernel, dispatchData, params.engineInfo, kernelName, jit, entry_point);
+    if (orgParams.outputs.size() == 1) {
+        FillCLKernelData(kernel, dispatchData, params.engineInfo, kernelName, jit, entry_point);
+    } else {
+        FillCLKernelData(kernel, dispatchData, params.engineInfo, kernelName, jit, entry_point,
+                         "", false, false, 1, GetFusedPrimitiveInputsCount(params), 2);
+    }
 
-    if (orgParams.outputs_num == 2) {
+    if (orgParams.outputs_num == 2 && orgParams.outputs.size() == 1) {
```

Comment:
please use a common function returning whether it is using multiple ouputs or not, so that we can refactor easily in the future

Reply:
done